### ISSUE TITLE
problem installing tensorflow

### ISSUE THREAD

**Issue by Pamulapati13** [hengluchang/newsum#6]
I am using higher version of tensorflow.But i am unable to install mentioned tensorflow version

**Comment by hengluchang** [comment]
There are two ways around this. First you can uninstall your current version of Tensorflow and install the older version same as this repo. The recommended way is to create a python virtual environment and install requirements in your virtual environment.

**Comment by Pamulapati13** [comment]
i am using gitbash and tensorflow 0.12 is causing many errors. is there a chance to run that on latest tensorflow versions

**Comment by Pamulapati13** [comment]
AttributeError: module 'tensorflow.tools.api.generator.api.nn' has no attribute 'seq2seq'
I even got this error

**Comment by xin527773783** [comment]
I solve this problem by using Anaconda，you can  create a python3.6  environment and install requirements in your environment.

**Comment by abhineet99** [comment]
> AttributeError: module 'tensorflow.tools.api.generator.api.nn' has no attribute 'seq2seq'
> I even got this error

Yeah they have shifted se2seq to legacy, check updates on tensorflow git.
